### DISCUSSION TITLE
Export to Excel

### DISCUSSION THREAD

**Discussion by chadgriff** [Budibase/budibase#2907] [Help] [answered]
Looking to learn how to configure a solution within Budibase to export the contents of a table component to a .CSV file.

**Comment by mjashanks** [comment, marked as answer]
Hey Chad. We do not have this capability at the moment, but it's been asked for before. Give it an upvote: https://github.com/Budibase/budibase/discussions/1885

**Reply by chadgriff** [reply]
Okay...I will close this ticket and monitor #1885 for progress on thissubject.
